SPECS/xmvn/xmvn.spec

```diff
@@ -153,10 +153,17 @@ mver=$(sed -n '/<mavenVersion>/{s/.*>\(.*\)<.*/\1/;p}' \
 mkdir -p target/dependency/
 cp -a "${maven_home}" target/dependency/apache-maven-$mver
 
+# Workaround easymock incompatibility with Java 17that should be fixed            
```

Comment:
if you brought these changes from fedora or other distros can you credit them in the changelog. Thanks